crates/antelope/src/api/client.rs

```diff
@@ -40,20 +39,6 @@ pub struct APIClient<P: Provider> {
 }
 
 impl<P: Provider> APIClient<P> {
-    pub fn default_provider(base_url: String) -> Result<APIClient<DefaultProvider>, String> {
```

Comment:
This library is used already by the snarktor POC, I think we should leave this constructor method so we don't break that POC.  Also I think this method is an easier way to construct it than using the `custom_provider` method.  But we should remove the _debug method.

Reply:
Code updated accordingly